packaging/windows/BUILD.txt

```diff
@@ -0,0 +1,106 @@
+How to make a darktable windows installer (64 bit Intel only):
+
+1). Install MSYS2 (instructions and prerequisites can be found on official website: https://msys2.github.io/).
+    
+    Install x64 developer toolchain from the MSYS2 prompt:
+    $ pacman -S mingw-w64-x86_64-toolchain
+    
+    Install required dependencies:
+    $ pacman -S mingw-w64-x86_64-{exiv2,lcms2,lensfun,libglade,dbus-glib,libusb-compat,openexr,sqlite3,libxslt,libsoup,libwebp,libsecret,lua,graphicsmagick,openjpeg2,gtk-engines,gtk3,pugixml,sdl,libexif}
+    
+    For osm-gps-map:
+    $ pacman -S mingw-w64-x86_64-{python2-cairo,python2-pygtk,osm-gps-map}
+
+    For gphoto2:
+    $ pacman -S mingw-w64-x86_64-{libusb,popt,libgphoto2}
+    Check that CAMLIBS and IOLIBS for LIBGPHOTO are available
```

Comment:
You can check it by "echo $CAMLIBS". I made a note here as normal MSYS2 packages do not like to add env variables, but libgphoto2 needs that. So for the current mingw-w64-x86_64-libgphoto2 I have included a script which adds this to the MSYS2 environment, but this comment is more of a reminder for later that the installer needs to take care of this aspect.